src/fallbackServiceStub.ts

```diff
@@ -97,6 +99,7 @@ export function generateServiceStub(
       for (const key of Object.keys(options)) {
         headers[key] = options[key][0];
       }
+      const streamArrayParser: StreamArrayParser = new StreamArrayParser(rpc);
```

Comment:
nit: no need to explicitly write its type `: Something` if it's assigned to `new Something()`.  TypeScript will deduct it.